cmd/markdown/atmos_git_usage.md

```
@@ -0,0 +1,11 @@
+- Manage Git repositories as Atmos artifacts
+
+```shell
+ $ atmos git clone <name-or-uri>
+ $ atmos git pull <name-or-path>
+ $ atmos git status <name-or-path>
+ $ atmos git diff <name-or-path>
+ $ atmos git commit <name-or-path> --message "msg"
+ $ atmos git push <name-or-path>
+ $ atmos git clean <name>
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)